frontend/src/components/TimeLogsPage/TimeLogForm.js

```diff
@@ -20,10 +24,34 @@ export const TimeLogForm = ({ handleSubmit, disabled }) => {
     setDescription(event.target.value)
   }
 
+  const formIsInvalid = () => {
+    let error_exists = false
```

Comment:
This one's also not needed; we can simplify the logic for example by returning `!!timeErrorMessage || !! descriptionErrorMessage`

Reply:
I tried returning e.g. (!!timeErrorMessage || !! descriptionErrorMessage) at the end of the formIsInvalid function, but it kept returning wrong values. I wonder if it has something to do with the errorMessages being checked before they're updated?

Reply:
Might have to do with React's asynchronous operations too